.config/.cprc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What exactly is this file for? I tried googling `.cprc.json` but couldn't find what it's for

Reply:
It's a file generated by create-plugin, I think it might be how plugin tools trace the version of create-plugin for upgrades etc.